content/doc/book/installing/index.adoc

```diff
@@ -1,79 +1,334 @@
-For other scenarios, see link:/doc/book/appendix/advanced-installation/[Advanced Jenkins Installation]
 
-== Installation
+=== Docker
+
+As described on the https://docs.docker.com/engine/docker-overview/[Docker
+overview] page, Docker is a platform for running applications in an isolated
+environment called a "container" (or Docker container). Applications like
+Jenkins can be downloaded as read-only "images" (or Docker images), each of
+which is run in Docker as a container. Therefore, a Docker container is in
+effect a "running instance" of a Docker image. From this perspective, an image
+is stored permanently more or less (i.e. insofar as image updates are
+published), whereas containers are stored temporarily. Read more about these
+concepts in the Docker documentation's
+https://docs.docker.com/get-started/[Getting Started, Part 1: Orientation and
+setup] page.
+
+Docker's fundamental platform and container design means that a single Docker
+image (for any given application like Jenkins) can be run on any operating
+system (e.g. macOS, Linux and Windows) which is also running Docker.
+
+
+==== Installing Docker
+
+To install Docker on your operating system, visit the
+link:https://www.docker.com/[Docker] website and navigate to *Get Docker* at the
+top of their site to begin accessing the Docker download that's suitable for
+your operating system. Follow the installation instructions on their website.
+
+*Note:* If you are installing Docker on a Linux-based operating system, ensure
+you configure Docker so it can be managed as a non-root user. Read more about
+this in Docker's
+https://docs.docker.com/engine/installation/linux/linux-postinstall/[Post-installation
+steps for Linux] page of their documentation. This page also contains
+information about how to configure Docker to start on boot.
+
+
+==== Downloading and running Jenkins in Docker
+
+There are several Docker images of Jenkins available.
+
+However, the recommended Docker image to use is the
+link:https://hub.docker.com/r/jenkinsci/blueocean/[`jenkinsci/blueocean` image].
+This image contains the current link:/download[Long-Term Support (LTS) release
+of Jenkins] (which is production-ready) bundled with all Blue Ocean plugins and
+features. This means that you do not need to install the Blue Ocean plugins
+separately.
+
+*Note:* A new `jenkinsci/blueocean` image is published each time a new release
+of Blue Ocean is published.
+
+
+*To download and run Jenkins with Blue Ocean in Docker:*
+
+. Open up a terminal/command prompt window.
+. Download the `jenkinsci/blueocean` image and run it as a container in Docker
+  using the following
+  link:https://docs.docker.com/engine/reference/commandline/run/[`docker run`]
+  command:
+** *On macOS and Linux:*
++
+[source,bash]
+----
+docker run \
+  --rm \  # <1>
+  -d \ # <2>
+  -p 8080:8080 \ # <3>
+  -v jenkins-data:/var/jenkins_home \ # <4>
+  -v /var/run/docker.sock:/var/run/docker.sock \
+  jenkinsci/blueocean # <5>
+----
```

Comment:
File creating an annotated/unannotated source block switch similar to `[pipeline]`.

Reply:
I've already made a note of this and will create a JIRA either later tonight or Monday.

Reply:
Fyi - https://issues.jenkins-ci.org/browse/WEBSITE-438 - sorry @bitwiseman - this was a bit late.